src/calculators/priming.rs

```diff
@@ -1,48 +1,53 @@
-use std::error::Error;
-use futures::{future, Future, stream, Stream};
-
 pub struct Priming;
 
 impl Priming {
+    pub fn celsius_to_fahrenheit(&self, celsius: f64) -> f64 {
```

Comment:
this function isn't needed anymore I guess.

Reply:
I've removed it, and the tests for `celsius_to_fahrenheit`.